### PR TITLE
feat(new-design): desktop version completed

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**

### CODE REVIEW

**Comment by Shaharyar-saleem** [comment]
Add to metamask is left.

**Comment by Shaharyar-saleem** [comment]
This is the desktop version with all the functionality of connect wallet.

File: assets/js/connectWallet.js [hunk 1]
```diff
@@ -643,35 +643,34 @@ async function initContract() {
...
-    // clipboard input value
-    const copyLink = document.getElementById("addressInput");
-    copyLink.value = user.address;
   } else {
     beginLogins();
   }
 }
 
 async function logOut() {
+  console.log("logout is pressed")
```
Comment (bobachef) [inline assets/js/connectWallet.js:673]: remove

File: assets/js/connectWallet.js [hunk 2]
```diff
@@ -643,35 +643,34 @@ async function initContract() {
 async function startUp() {
   if (user.address != undefined) {
     let p2 = user.address.slice(42 - 5);
-    $("#shortAddress")[0].innerHTML = `${user.address.slice(0, 4)}...${p2}`;
     $("#shortAccount")[0].innerHTML = `${user.address.slice(0, 4)}...${p2}`;
     $("#showAccountBtn")[0].innerHTML = `${user.address.slice(0, 4)}...${p2}`;
 
     // Display Network Error
     const chainId = await web3.eth.getChainId();
+    console.log(chainId, "this is the chain id")
```
Comment (bobachef) [inline assets/js/connectWallet.js:651]: remove

File: assets/js/connectWallet.js (hunk 1 above)
Comment (Shaharyar-saleem) [reply assets/js/connectWallet.js:673]: removed

File: assets/js/connectWallet.js (hunk 2 above)
Comment (Shaharyar-saleem) [reply assets/js/connectWallet.js:651]: removed

**Pull request merged**